inmanta-module-{{cookiecutter.module_name|slugify}}/tests/test_basics.py

```diff
@@ -1,4 +1,6 @@
-{%- if cookiecutter.license == "ASL 2.0" -%}
+from pytest_inmanta.plugin import Project
```

Comment:
Shouldn't this go below the file header?  This is usually how we do, not sure if this is by habit or convention, but it would look more consistent in any case.

Reply:
No idea, i'll change it.